src/ParallelAlgorithms/Interpolation/Callbacks/SendGhWorldtubeData.hpp

```diff
@@ -32,7 +33,7 @@ namespace callbacks {
 ///
 /// \note This callback requires the temporal ID in an InterpolationTargetTag be
 /// a TimeStepId.
-template <typename CceEvolutionComponent, bool DuringSelfStart>
+template <typename CceEvolutionComponent, bool DuringSelfStart, bool LTS>
```

Comment:
I'd think both `DuringSelfStart` and `LTS` are wanted because we need to distinguish self-start from evolution even when the interface manager is used (`Cce::Actions::ReceiveGhWorldtubeData` takes `DuringSelfStart` other than `NeedsInterfaceManager`).

Reply:
Ahh I see, then I think this `LTS` template is ok.